assets/api-swagger/swagger.json

```diff
@@ -914,6 +914,228 @@
+        "parameters": [
+          {
+            "name": "Ometria-Account-Id",
+            "in": "header",
+            "description": "ID for the account in which to search for the profiles.",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "limit",
+            "in": "query",
+            "description": "The limit for pagination purposes.",
+            "required": false,
+            "schema": {
+              "type": "integer",
+              "minimum": 1,
+              "maximum": 100,
+              "default": 10
+            }
+          },
+          {
+            "name": "offset",
+            "in": "query",
+            "description": "The offset for pagination purposes.",
+            "required": false,
+            "schema": {
+              "type": "integer",
+              "minimum": 0,
+              "default": 0
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The statuses of the merge jobs.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "offset": {
+                      "type": "integer",
+                      "description": "The offset used for pagination."
+                    },
+                    "limit": {
+                      "type": "integer",
+                      "description": "The limit used for pagination."
+                    },
+                    "job_count": {
+                      "type": "integer",
+                      "description": "The number of merge jobs returned."
+                    },
+                    "total_job_count": {
+                      "type": "integer",
+                      "description": "The total number of merge jobs in the account."
+                    },
+                    "jobs": {
+                      "type": "array",
+                      "items": {
+                        "$ref": "#/components/schemas/MergeProfileJobStatus"
+                      }
+                    }
+                  }
+                },
+                "example": {
+                  "offset": 2,
+                  "limit": 10,
+                  "job_count": 3,
+                  "total_job_count": 5,
+                  "jobs": [
+                    {
+                      "job_id": 12,
+                      "status": "finished"
+                    },
+                    {
+                      "job_id": 13,
+                      "status": "failed"
+                    },
+                    {
+                      "job_id": 14,
+                      "status": "in_progress"
+                    }
+                  ]
+                }
+              }
+            }
+          },
+          "400": {
```

Comment:
Do we need to include here the 403 response?

Reply:
I don't see why not if it's present in the other endpoints.